src/server/db/schema.ts

```diff
@@ -68,6 +68,19 @@ export const club = pgTable('club', {
   image: text('image').default('/nebula-logo.png').notNull(),
 });
 
+export const events = pgTable('events', {
+  id: integer('id')
+    .default(sql`nanoid(20)`)
+    .primaryKey(),
+  clubId: text('club_id')
+    .notNull()
+    .references(() => club.id, { onDelete: 'cascade' }),
+  name: text('name').notNull(),
+  description: text('description').notNull(),
+  startTime: timestamp('start_time').notNull(),
+  endTime: timestamp('end_time').notNull(),
```

Comment:
Try using this with the timezone included just to make sure that nothing is messed up when different timezones are involved. I know right now its not a huge issue but it could become a bigger issue if we don't change it now